src/components/DataViewPrimaryFilter.jsx

```diff
@@ -33,12 +33,17 @@ const LabelWrapper = styled.div`
 		background-color: ${theme.colors.blue}45;
 	}
 
-	& > .filter {
+	& > .filter-label {
 		font-weight: ${theme.fontWeight.normal};
 		margin-right: 8px;
 		text-transform: capitalize;
 	}
 
+	& > .filter-value {
+		font-weight: ${theme.fontWeight.normal};
+		margin-right: 8px;
+	}

```

Comment:
This should have a "text-transform: none" because then it doesn't actually do what the ticket asks. Currently we have a text transform: uppercase on all buttons, but on the filters, we want our values to render just as the user writes them.